### DISCUSSION TITLE
Miles default for meeting search

### DISCUSSION THREAD

**Discussion by nataliefaith2** [code4recovery/12-step-meeting-list#1218] [Q&A] [answered]
I have had the meeting setting for meetings to kilometres since set up, yet when selecting "Near Me" the meetings display in miles distance instead of kilometres. I noticed in the code miles is set as default - which is no doubt great for US & UK meetings but in Australia, we use "km".

So I want to know how to change this default in the code so our members see km/ kilometres when looking for meeting "near me" or look up distance.

See the attached screenshots to explain the above.

<img width="1220" alt="Screen Shot 2023-09-19 at 11 10 55 am" src="https://github.com/code4recovery/12-step-meeting-list/assets/23748349/c329095e-2b87-47fa-b6d8-871adac6c74b">

<img width="537" alt="Screen Shot 2023-09-19 at 11 11 27 am" src="https://github.com/code4recovery/12-step-meeting-list/assets/23748349/b10dd125-dd3b-40ac-a1dc-08aa48bbae6a">

If you'd like to test this yourself, [go to the test site]([url](https://staging.al-anon.org.au/find-a-meeting/):
[
](https://staging.al-anon.org.au/find-a-meeting/)

_Please ensure your location services are turned on, so the geolocation will be able to show you which meetings are closest to you by distance. This is the only time it appears in miles._

**Comment by joshreisner** [comment, marked as answer]
hi @nataliefaith2 

when you use TSML UI at the default `/meetings` page the distance units setting will get passed through, but in your case you are embedding TSML UI on a page using an embed code, which is fine, but means you must specify the distance units using the instructions: https://github.com/code4recovery/tsml-ui/#use-kilometers

but looks like you found them though? where you have

```
<div var tsml_react_config = {
  distance_unit: 'km',
};></div>
```
just change that to
```
<script>
var tsml_react_config = {
  distance_unit: 'km',
};
</script>
```
and you should be all right

**Reply by nataliefaith2** [reply]
Awesome, thank you :)
